src/mmw/js/src/core/modals/views.js

```diff
@@ -388,8 +389,34 @@ var PlotView = ModalBaseView.extend({
     }
 });
 
+var CreateAccountPersuasionModalView = ModalBaseView.extend({
```

Comment:
`CreateAccountModal` is too short :) Just kidding, let's do it! I didn't go with it originally because it seemed too similar to `SignUpModal`, but the alternatives are a bit ridiculous.